ssz/sedes/serializable.py

```diff
@@ -169,21 +170,41 @@ def copy(self, *args, **kwargs):
         ).difference(
             self._meta.field_names[:len(args)]
         )
-        unchanged_kwargs = {
-            key: copy.deepcopy(value)
-            for key, value
-            in self.as_dict().items()
-            if key in missing_overrides
-        }
+        unchanged_kwargs = {}
+        for key, value in self.as_dict().items():
+            if key in missing_overrides:
+                is_immutable_field = (
+                    isinstance(value, BASE_TYPES) or
```

Comment:
it might be faster here to do `if type(value) in BASE_TYPES` as `isinstance` checks can be expensive and we have no need to support subclasses of `int`, `bool`, or `bytes` that I know of.  The same could go for the `isinstance(value, tuple)` check below.